README.md

```diff
@@ -23,14 +25,22 @@ Token Server is exposed as resource. So in this example the Token Server acts as
-| device.api.serverRoot                     | The base uri of the resource server (Token Server in this example).                                                                                                                                            |
-| device.api.username                       | The basic authentication username used to access the api.                                                                                                                                                      |
-| device.api.password                       | The basic authentication password used to access the api.                                                                                                                                                      |
+Refer to the Spring
+Boot [Externalized Configuration](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config) for
+setting the properties.
+
+The following properties can be set for the Example Resource Gateway:
+
+| Property                                  | Default value | Example value               | Description
+|-------------------------------------------|---------------|-----------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+| resource.gateway.tokenserver.baseUri      |               | http://localhost:7878/oauth | The base URI of the [Onegini Token Server](https://docs.onegini.com/msp/stable/token-server).
+| resource.gateway.tokenserver.clientId     |               |                             | The client id of the API client acting as resource gateway, see [Resource Gateway config](https://docs.onegini.com/msp/stable/token-server/topics/general-app-config/resource-gateway/resource-gateway.html).
+| resource.gateway.tokenserver.clientSecret |               |                             | The client secret of the API client acting as resource gateway.
+| device.api.serverRoot                     | The value of resource.gateway.tokenServer.baseUri | http://localhost:7878/oauth | The base uri of the resource server that returns a list of Devices. In this example, it is the base URI of the Onegini Token Server.
+| device.api.username                       |               |                             | The basic authentication username used to [access the End User api](https://docs.onegini.com/msp/stable/token-server/topics/technical-app-management/api-configuration/api-configuration.html) in the Onegini Token Server.
+| device.api.password                       |               |                             | The basic authentication password used to access the End User api in the Onegini Token Server.
+| spring.servlet.multipart.max-file-size    | 1MB           |                             | Sets the maximum file size per uploaded file.
```

Comment:
Is 500 status code ok to return in case we try to upload a bigger file?

Reply:
Depends on your point of view:
* 400: it's the client's fault that it sends something big retry again with a different request
* 500: it's the server's problem that it cannot handle something big, but with a different setting it might accept it in the future

Reply:
Yeah, I think it is fine as it points to a potential application container configuration issue.